### DISCUSSION TITLE
Variable font support

### DISCUSSION THREAD

**Discussion by nfon-cm** [six7/figma-tokens#1785] [Q&A] [answered]
Hi, I was wondering how to go about setting the axes for variable fonts through the typography tokens?
Do we have access to each axis?

<img width="423" alt="image" src="https://user-images.githubusercontent.com/128799722/229536479-54506300-daa9-47df-8b2d-e245a8e010b6.png">

Thanks!

**Comment by six7** [comment]
Unfortunately Figma's Plugin API doesn't support this yet.

**Reply by nfon-cm** [reply]
I see. Do you have any idea if it's planned?

**Reply by six7** [reply, marked as answer]
No, only the Figma gods know unfortunately 🔮
